src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.cs

```diff
@@ -276,17 +276,23 @@ public sealed partial class ECDsaCng : System.Security.Cryptography.ECDsa
         public ECDsaCng() { }
         public ECDsaCng(int keySize) { }
         public ECDsaCng(System.Security.Cryptography.CngKey key) { }
+#if !netfx // types missing from netfx targeting pack
```

Comment:
@weshaggard any idea if the desktop targeting pack is going to include these at some point?